lib/ansible/modules/web_infrastructure/jenkins_job.py

```diff
@@ -216,7 +216,7 @@ def get_job_status(self):
             if "color" not in response:
                 return self.EXCL_STATE
             else:
-                return response['color'].encode('utf-8')
+                return str(response['color'])
```

Comment:
Could you please use `to_native` or `to_text` instead? You can find the function [here](https://github.com/ansible/ansible/blob/875ac4b7a52f140c331fbd2ac59b3ccd3645765a/lib/ansible/module_utils/_text.py#L169)

Reply:
@Akasurde Thank you very much for the feedback! Changed to `to_native`.